pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -547,4 +547,10 @@ stdenv.mkDerivation ({
 // optionalAttrs (!stripped || cross != null) { dontStrip = true; NIX_STRIP_DEBUG = 0; }
 
 // optionalAttrs (enableMultilib) { dontMoveLib64 = true; }
+
+# This is only here because it was here in previous commits and we
+# want to avoid a meaningless mass rebuild.  These variables get
+# overridden via crossAttrs in an actual cross-build.
+// { crossMingw = false; crossStageStatic = true; }
+
```

Comment:
This can be merged with an update that requires a mass-rebuild on it's own so the extra cost could be minimal.
This line needs to be duplicated into every gcc expression that should be asserted inside the tests, right? Imo that's an even stronger argument for moving it inside the tests file and make it generic.
For me it's as simple as: If you only need it for the tests, put it inside the tests file.